projects/hr-portal/src/index.css

```diff
@@ -0,0 +1,17 @@
+html, body {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+  
+  body {
+    display: flex;
+    font-family: 'Helvetica', sans-serif;
+  }
+  
+  app-hr-portal {
+    flex: 1;
```

Comment:
These override the `:host` styles that you set in the `hr-portal.ts` file. If you dont need this, please remove it. Also, why flex on a host that has only 1 element?